### PR TITLE
Capstone Project Microverse module 1

### PR DESCRIPTION
**Pull request by lucas-crodrigues**
This is the first capstone project in my Microverse journey.

For this project, we were required to follow the template provided [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), but fill it with the content of our choosing.

My content is fictional but based on [this seminar](https://institutoayrtonsenna.org.br/pt-br/seminario-internacional-motivacao.html) by [Instituto Ayrton Senna](https://institutoayrtonsenna.org.br/en.html).

Here is the link to the [Loom video](https://www.loom.com/share/4cbc3ed299894b859ee2a835471e003b) presenting the code.

### CODE REVIEW

**Comment by lucas-crodrigues** [comment]
As you can see I'm getting those "Error: Process completed with exit code 1." on my linters, apart from that they're all passing.

File: scripts.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+              <div></div>
+              <p>${speakersDetails[5].description}</p>
+            </div>
+          </div>
+        </div>
+        <button class="showMore" onclick="showSpeakers();">MORE ▼</button>`;
+}
+
+function showSpeakers() { // eslint-disable-line no-unused-vars
+  const showMore = document.querySelector('.showMore');
```
Comment (Hamzaoutdoors) [inline scripts.js:131]: - [OPTIONAL] Great job implementing the `show more` functionality in the speaker's section, I am recommending you add the `show less` button that will decreament the speaker's cards ^_^

design

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @lucas-crodrigues , I am **Hamza** your reviewer for this PR 😸 

_Good job so far! I enjoyed surfing your fancy website_
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

🔴  - [REQUIRED] You are asked in this project to record a **5min** that contains a demo of your project, showing the pages in different screen sizes and some highlights about a piece of the code that you find interesting and proud of, Kindly record a video and give the link in your README and inside you Pull Request description.

🔴 - [REQUIRED] As required in this capstone project, your project should've deployed using the `GitHub Pages` and add the link in your Readme file, Kindly add a deployment to this amazing web page and let the world explore it 😄 

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @hamzaoutdoors in your question so I can receive the notification ^_^

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    picture: 'Art/speaker_06.png',
+    name: 'William Kamkwamba',
+    occupation: 'Inventor and bestselling author',
+    description: 'William recounts his story in his memoir The Boy Who Harnessed the Wind: Creating Currents of Electricity and Hope',
+  },
+];
+
+const featuredSpeakers = document.querySelector('.featuredSpeakers');
+
+function loadSpeakers() { // eslint-disable-line no-unused-vars
```
Comment (danmainah) [inline scripts.js:63]: - [ ] [OPTIONAL] Kindly try your best to avoid disabling linters in your code, its not good practice to disable linters.

**Review by danmainah — APPROVED** [review]
Hi @lucas-crodrigues , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**